core/chains/evm/log/exports_test.go

```diff
@@ -0,0 +1,12 @@
+package log
```

Comment:
Is `exports_test.go` the same as `helpers_test.go` or do they do something different?

Reply:
Ya I created this file just to hold exports for testing.
I preferred putting exports in a separate file. But ok if people prefer to put all such stuff inside the helpers_test.go file too.